packages/stack/blockchain-client/src/index.js

```diff
@@ -19,6 +20,16 @@ class BlockchainClient {
         if (err) {
           return cb(err);
         }
+        if (endpoint.startsWith('ws')) {
+          return cb(null, new Web3.providers.WebsocketProvider(endpoint, {
+            headers: {Origin: constants.embarkResourceOrigin},
+            // TODO remove this when Geth fixes this: https://github.com/ethereum/go-ethereum/issues/16846
+            //  Edit: This has been fixed in Geth 1.9, but we don't support 1.9 yet and still support 1.8
+            clientConfig: {
+              fragmentationThreshold: 81920
+            }
+          }));
```

Comment:
Not sure what it takes to support Geth 1.9, but either way, I guess once we reach that points and/or the issue in the above is fixed, we don't just want to drop support for geth < 1.9 right? 

Maybe we can detect what geth version we're dealing with and only then set the threshold?